exec/cnex/cnex.c

```diff
@@ -19,16 +19,30 @@
 #include "cell.h"
 #include "dictionary.h"
 #include "exec.h"
+#include "extension.h"
 #include "global.h"
 #include "framestack.h"
 #include "module.h"
+#include "neonext.h"
 #include "nstring.h"
 #include "number.h"
 #include "opcode.h"
+#include "rtl_platform.h"
 #include "stack.h"
 #include "support.h"
 #include "util.h"
 
+#ifdef _WIN32
+#define PATHSEP "\\"
+#define LIBRARY_NAME_PREFIX ""
+#else
+#define PATHSEP "/"
+#define LIBRARY_NAME_PREFIX "lib"
+#endif
+
+TExtensionModule *g_ExtenstionModules;
+extern struct Ne_MethodTable ExtensionMethodTable;
```

Comment:
You declare this `extern` here, but then provide a definition in this module later. Which is it?

Reply:
Removed extern decl.